Test plugin with multiple versions of nexus

talebipour opened the pull request:
+ Provide feature to test plugin with multiple versions of nexus
 + Test plugin with nexus 3.24.0

Commit:
Test plugin with multiple versions of nexus

 + Provide feature to test plugin with multiple versions of nexus
 + Test plugin with nexus 3.24.0

MSDehghan commented on pom.xml:
```diff
@@ -108,6 +114,8 @@
                 <configuration>
                     <instructions>
+                        <!-- Relax dependency version in order to allow run inside multiple versions of nexus -->
+                        <_consumer-policy>$(version;==)</_consumer-policy>
```
i think it should be like this:
```suggestion
                        <_consumer-policy>$${version;==}</_consumer-policy>
```

talebipour replied:
Done

MSDehghan commented on src/test/java/ir/sahab/nexus/plugin/tag/IntegrationTest.java:
```diff
@@ -268,6 +272,7 @@ private void assertDefinitionEquals(TagDefinition expected, Tag actual) {
 
     @AfterClass
     public static void tearDown() {
+        repositoryRule.after();
```
Why not define RepositoryRule as a ClassRule??

talebipour replied:
It's already a rule, I was testing sth and I forgot to remove it.

Commit:
Few fixes + Prepare release 1.0.2

+ Change consumer policy format
+ Fix validation message for component criterioon

MSDehghan reviewed (approved):
it's OK
